restricted_site_access.php

```diff
@@ -780,7 +780,14 @@ public static function page_cache_notice() {
 			?>
 			<div data-rsa-notice="page-cache" class="notice notice-error is-dismissible">
 				<p>
-					<strong><?php esc_html_e( 'Page caching appears to be enabled. Restricted Site Access may not work as expected. <a href="https://wordpress.org/plugins/restricted-site-access/#faq">Learn more</a>.', 'restricted-site-access' ); ?></strong>
+					<strong><?php
+						echo wp_kses_post(
+							sprintf( '%s <a href="https://wordpress.org/plugins/restricted-site-access/#faq">%s</a>.',
```

Comment:
Don't concatenate translated strings 😄 

Let's follow WordPress core's example here:

https://github.com/WordPress/WordPress/blob/8662d9a74d67669da86e2f5181f1f972229648e8/wp-admin/admin-footer.php#L35-L39

This allows the plugin URL to be translated as well as generally allow for proper translation of the entire string (think RTL languages).

Reply:
Ah, ok - I'll change it to that approach. For some reason I was considering removing the html from the translation strings more important.  Makes sense for the context though.